swarm/swarm.go

```diff
@@ -51,19 +57,29 @@ func run(cmd string) {
 }
 
 func getleaderHost() string {
```

Comment:
@smothiki the code that I changed in the `getleaderHost` obtains the machines and the leader. I ask for a review of those changes